add del account docs (#134)
deta/space-docs: src/content/docs/use/your-account.md

---
title: Your Account
layout: "@layouts/DocsPageLayout.astro"
---

> # Deleting Your Account 

Reviewer: ```suggestion
## Deleting Your Account 
```

> You can submit a request to delete your account any time. We will delete your account and data within 30 days of receiving your request.

Reviewer: ```suggestion
You can submit a request to delete your account any time. After receiving your request, an automated job will delete your account and data within 30 days.
```

> In order to delete your account, login to your [Space Dashboard](https://deta.space), open the Teletype (command bar), click on **Settings** and navigate to the **Account** tab.

Reviewer: ```suggestion
In order to delete your account, login to your [Space Dashboard](https://deta.space), open [Teletype](/docs/en/use/interface#teletype), click on **Settings** and then navigate to the **Account** tab.
```

> Deleting your account will delete all your data, including your collections and installed apps. You will not be able to recover your data after deleting your account. 

Reviewer: ```suggestion
Deleting your account will delete all your data, including your [Collections](/docs/en/use/your-data/collections) and all [apps](/docs/en/use/space-apps) you've installed. You will not be able to recover your data after deleting your account. 
```

> If you are a developer and have published apps on Deta Space, [learn what happens to your apps after deleting your account](#your-published-apps-after-deleting-your-account).

Reviewer: ```suggestion
```

![delete-account](/docs_assets/use/account-deletion-1.png)

> ## Your published apps after deleting your account

Reviewer: ```suggestion
### Your published apps
```

If you have published an app on Deta Space that has been installed by anyone else other than yourself, _the app will continue to be available on Deta Space even after you delete your account_.

> This is to ensure that the app continues to be available to users who have installed it.

Reviewer: ```suggestion
This is to ensure that the app continues to work for users who have installed it.
```

> If the app was a listed app on Discovery, it will be unlisted after you delete your account. For the existing app release page on Discovery, we will add a note that the app is no longer maintained.

Reviewer: ```suggestion
If your published app was [listed](/docs/en/publish/releasing#releases), it will be [unlisted](/docs/en/publish/releasing#releases) after you delete your account. For the existing app [Discovery page](/docs/en/use/space-apps/discovery#app-discovery-pages), we will add a note that the app is no longer maintained.
```

> If no user has installed the app, the app will be also be deleted.

Reviewer: ```suggestion
If no user has installed your app, it will be deleted.
```
